packages/examples/packages/settings-page/src/index.tsx

```diff
@@ -0,0 +1,19 @@
+import type { OnSettingsPageHandler } from '@metamask/snaps-sdk';
+import { Box, Heading, Text } from '@metamask/snaps-sdk/jsx';
+
+/**
+ * Handle incoming settings page requests from the MetaMask clients.
+ *
+ * @returns A static panel rendered with custom UI.
+ * @see https://docs.metamask.io/snaps/reference/exports/#onsettingspage
+ */
+export const onSettingsPage: OnSettingsPageHandler = async () => {
+  return {
+    content: (
+      <Box>
+        <Heading>Hello world!</Heading>
+        <Text>Welcome to my Snap settings page!</Text>
```

Comment:
Can we add a toggle or something to show how one would build a settings page and save the data from it?

Reply:
Coming soon :tm: